.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Build and publish python package
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  publish-service-client-package:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Publish PyPi package
+        uses: code-specialist/pypi-poetry-publish@v1
+        with:
+          ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PUBLISH_REGISTRY_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```

Comment:
What's the scope of the `PYPI_TOKEN`? Might be worth a comment just to ensure it is kept minimal (since I imagine we must have an EESSI org on PyPI?).

Reply:
The scope is specific to the project. :)